pkg/common/cns-lib/volume/manager.go

```diff
@@ -557,6 +557,11 @@ func (m *defaultManager) DeleteVolume(ctx context.Context, volumeID string, dele
 		}
 		volumeOperationRes := taskResult.GetCnsVolumeOperationResult()
 		if volumeOperationRes.Fault != nil {
+			_, ok := volumeOperationRes.Fault.Fault.(cnstypes.CnsVolumeNotFoundFault)
```

Comment:
I do not see CnsVolumeNotFoundFault is present in vSphere 67u3. Can we make sure to use the fault present in all vSphere releases? Which vSphere release this fault is introduced?

Reply:
Can you try 

> _, isNotFoundFault := volumeOperationRes.Fault.Fault.(*vim25types.NotFound)

Reply:
Changed